src/main/java/net/minecraftforge/registries/IForgeRegistry.java

```diff
@@ -48,7 +49,12 @@
     @Nullable ResourceLocation getKey(V value);
 
     @Nonnull Set<ResourceLocation>           getKeys();
+    @Deprecated // TODO: remove in 1.13
     @Nonnull List<V>                         getValues();
+    @Nonnull
+    default Collection<V>                    getValuesCollection() {
```

Comment:
Would it be an idea to add a TODO for 1.13 to rename this back to `getValues`?

Reply:
Maybe, that might just be annoying to have people change and then change back though.